### ISSUE TITLE
72000 frames limit

### ISSUE THREAD

**Issue by svanterwingen** [epoc-ed/GUI#79]
I was wondering why there is a limit of 72000 frames for the viewer? After that number of frames you have to restart it.

**Comment by erikfrojdh** [comment]
Yes, it resets every time you start either a live view or a data collection. 

The reason to have a limit is that Jungfraujoch (our data backend) allocates buffers depending on the amount of frames it expects to collect. Setting a much larger number will make the start and stop slower. 

If the 1h is an issue we could also put make it 2-3 we just have to check that start and stop still are fast.

**Comment by erikfrojdh** [comment]
No there is no issue leaving the detector on. It might even be good for thermal stability. I'll try to increase the timeout to 2-3h when we are in Vienna next week.
